src/FakeItEasy/Analysis/MustUseReturnValueAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace FakeItEasy.Analysis
+{
+    using System;
+
+    /// <summary>
+    /// Indicates that the return value of a method should not be ignored.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public sealed class MustUseReturnValueAttribute : Attribute

```

Comment:
> I did make the attribute internal and it all built.

Yes, because the analyzer has no compile-time reference to the attribute. It just uses Roslyn to look at the publicly visible attributes on the `IMethodSymbol`.